demos/astro-playground/sentry.client.config.ts

```diff
@@ -1,7 +1,21 @@
 import * as Sentry from '@sentry/astro';
+import { getCurrentHub, type Event } from '@sentry/astro';
 
 Sentry.init({
   debug: true,
   integrations: [new Sentry.BrowserTracing(), new Sentry.Replay()],
   tracesSampleRate: 1.0,
 });
+
+if (process.env.NODE_ENV === 'development') {
+  import('@spotlightjs/astro').then(Spotlight => {
```

Comment:
so we could actually add localStorage settings into spotlight and make this a default behavior for _any_ unhandled error coming in too.

can also keep this behavior (i think we NEED it) and continue w/ a setting to toggle it